app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -1,5 +1,6 @@
 require 'fog_extensions/vsphere/mini_servers'
 require 'foreman/exception'
+require 'rbvmomi'
```

Comment:
Is this always available?

Reply:
It's fog-vsphere's dependency

Reply:
yes, but do we always ship the gem in production? We have a [metapackage for vmware](https://github.com/theforeman/foreman-packaging/blob/3879d30037d099a48f8480e91cd0f029c883a71e/packages/foreman/foreman/foreman.spec#L554-L564), that might not be installed. It adds the [fog-vsphere dependency](https://github.com/theforeman/foreman/blob/develop/bundler.d/vmware.rb).